doc/user-guide-v1.adoc

```diff
@@ -1494,6 +1545,80 @@ To change this default configuration, see the following sections.
+
+For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/[Customizing DNS Service].
+
+NOTE: `Default` is not the default DNS policy. If `.spec.dns.policy` is not explicitly specified, then `ClusterFirst` is used.
+
+DNS Config allows users more control over the DNS settings for an application Pod.
+
+The `.spec.dns.config` field is optional and it can work with any `.spec.dns.policy` settings. However, when a `.spec.dns.policy` is set to `None`, the `.spec.dns.config` field must be specified.
+
+The following properties are specified within the `.spec.dns.config` field:
+
+* `.spec.dns.config.nameservers`: a list of IP addresses that are used as DNS servers for the Pod. Up to 3 IP addresses are specified. When `.spec.dns.policy` is set to `None`, the list must contain at least one IP address, otherwise this property is optional. The servers that are listed are combined to the base name servers generated from the specified DNS policy with duplicate addresses removed.
+* `.spec.dns.config.searches`: a list of DNS search domains for hostname lookup in the Pod. This property is optional. When specified, the provided list is merged into the base search domain names that are generated from the chosen DNS policy. Duplicate domain names are removed. Kubernetes allows up to 32 search domains.
+* `.spec.dns.config.options`: an optional list of objects where each object must have a name property and can have a value property. The contents in this property are merged to the options generated from the specified DNS policy. Duplicate entries are removed.
+[source,yaml]
+----
+spec:
+  dns:
+    policy: "None"
+    config:
+      nameservers:
+        - 192.0.2.1 # this is an example
+      searches:
+        - ns1.svc.cluster-domain.example
+        - my.dns.search.suffix
+      options:
+        - name: ndots
+          value: "2"
+        - name: edns0
+----
+
+For more information on DNS, see link:https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/[the Kubernetes DNS documentation].
+
+[[configure-tolerations]]
+=== Configure tolerations (.spec.tolerations)
+
+NOTE: Only available for operator version v1.4.0+
+
+Node affinity is a property that attracts pods to a set of nodes either as a preference or a hard requirement. However, taints allow a node to repel a set of pods.
+
+Tolerations are applied to pods and allow a scheduler to schedule pods with matching taints. The scheduler also link:https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/[evaluates other parameters] as part of its function.
+
+Taints and tolerations work together to help ensure that application pods are not scheduled onto inappropriate nodes. If one or more taints are applied to a node, the node cannot accept any pods that do not tolerate the taints.
+
+Tolerations can be configured in WebSphereLibertyApplication CR by using the `.spec.tolerations` field.
```

Comment:
Again unclear what operators you are talking about whether is Open Liberty or Websphere Liberty

Reply:
fixed